Feliz.AntDesign/Ant.fs

```diff
@@ -34,4 +34,16 @@ type Ant =
     static member inline icon (children: #seq<ReactElement>) =
         ofImport "Icon" "antd" children
     static member inline icon (properties: IReactProperty list) =
-        Interop.reactApi.createElement(import "Icon" "antd", createObj !!properties)
\ No newline at end of file
+        Interop.reactApi.createElement(import "Icon" "antd", createObj !!properties)
+    static member inline text (children: #seq<ReactElement>) =
+        ofImport "Typography.Text" "antd" children
```

Comment:
When using `ofImport`, the `children` parameter has to be the last one and before it is an empty list for the properties:
```fs
ofImport "Typography.Text" "antd" [ ] children
```